packages/lex-cli/src/codegen/lex-gen.ts

```diff
@@ -14,15 +14,18 @@ import {
   LexCidLink,
   LexBytes,
   LexIpldType,
+  LexXrpcParametersProperty,
+  LexRef,
 } from '@atproto/lexicon'
 import { toCamelCase, toTitleCase, toScreamingSnakeCase } from './util'
+import { PROPERTY_VALID_REF_TYPES } from '@atproto/lexicon/src/validators/xrpc'
```

Comment:
I wanna avoid imports like this, so wherever this lands, we should properly export it so we can just import from `@atproto/lexicon`

Reply:
Yeah, I've seen that before pushing, but I wasn't sure where it should even go, but having thought about it, it belongs in the xrpc validators file. I'll push an fix later

Reply:
Should be good now